docs/src/pages/components/switches/UseSwitchesMaterial.js

```diff
@@ -1,399 +0,0 @@
-/* eslint-disable no-restricted-imports, react/prop-types */
-import * as React from 'react';
-import clsx from 'clsx';
-import {
-  unstable_composeClasses as composeClasses,
-  useSwitch,
-} from '@material-ui/unstyled';
-import { alpha, darken, lighten, useThemeProps, styled } from '@material-ui/system';
-import { ThemeProvider, createTheme } from '@material-ui/core/styles';
-import { capitalize } from '@material-ui/core/utils';
-import {
-  useFormControl,
-  switchClasses,
-  getSwitchUtilityClass,
-} from '@material-ui/core';
-import TouchRipple from '@material-ui/core/ButtonBase/TouchRipple';
-import useTouchRipple from '@material-ui/core/useTouchRipple';
```

Comment:
Speaking of private API, how we handle the touch ripple seems off:

- TouchRipple is private
- useTouchRipple, getTouchRippleUtilityClass, touchRippleClasses are public

Shouldn't we make everything private or everything public?

Reply:
I wasn't aware this abstraction was created. It was yet another abstraction that was added while working on a specific problem. That's another reason why early abstractions should be avoided. It's just not viable to review the abstraction and the specific solution in one PR. Especially if that PR has hundreds of changed lines.